src/crud.js

```diff
@@ -43,10 +44,41 @@ const renderTasks = () => {
       }
     });
   });
-};
+
+  // Checkbox input
+  const checkBoxes = document.querySelectorAll('.checkbox');
+  checkBoxes.forEach((box, index) => {
+    box.addEventListener('change', () => {
+      if (box.checked) {
+        tasks[index].completed = true;
+      } else {
+        tasks[index].completed = false;
+      }
+      localStorage.setItem('tasks', JSON.stringify(tasks));
+    });
+  });
```

Comment:
- [ ] As stated in the project requirement Kindly make sure that you **Add a new JavaScript file and import it as a module: that contain methods related to the status updates (completed: true / false).**kindly Fix it.

Reply:
This change has been done well

Reply:
please expound